less/index.less

```diff
@@ -164,4 +188,24 @@ html, body {
 		border-bottom:1px solid #333333;
 
 	}
-}
\ No newline at end of file
+}
+
+.experience{
+	background-color:#FFFFFF;
+	max-width: 400px;
+	height:750px;
+	margin-bottom:25px; 
+	margin-top:50px;
+	padding-left: 50px;
+	color:#333333;
+	border-left: 4px solid #333333;
+
+
+	h3{
+		color:#333333;
+		font-size:2rem;
+		border-bottom:1px solid #333333;
```

Comment:
You can do a shorthand with only 3 characters for the hex colors like so: `#333`, `#FFF` 😉